### PR TITLE
small tweaks and added .JS file

### PR DESCRIPTION
**Pull request by dmryan711**

### CODE REVIEW

File: Home-page.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h1 class="text-center">Gang Out</h1>
       <div class="card" id="userdata" style="width: 18rem;">
         <img src="..." class="card-img-top" alt="...">
         <div class="card-body">
-          <p class="card-text">Franky Two Toes</p>
+          <p class="card-text">...</p>
```
Comment (dmryan711) [inline Home-page.html:28]: do we want to to add text here or will that come in a later PR?

File: logic.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+//Landing-page START
+var groupID;
```
Comment (dmryan711) [inline logic.js:2]: can we wrap this in a IFFE to prevent polluting the global scope?

**Review by dmryan711 — COMMENTED** [review]
🔥

File: Home-page.html (hunk 1 above)
Comment (tktommykline) [reply Home-page.html:28]: Yeah that will be added later

File: logic.js (hunk 2 above)
Comment (tktommykline) [reply logic.js:2]: lets talk on it later

Comment (dmryan711) [reply logic.js:2]: perfect

**Review by tktommykline — APPROVED** [review]
cool

**Pull request merged**